build-java/resources/templates/README.mustache

```diff
@@ -0,0 +1,104 @@
+# Criteo Marketing SDK for Java
```

Comment:
The new generated README.md makes much more sense !

As the java's README.mustache is copied from the openapi-generator, it would be great to create a separate commit just for this copy action. That will be much more easier to track that had been changed in the template.